source/docs/articles/wordpress/importing-a-wordpress-site.md

```diff
@@ -6,53 +6,122 @@ category:
-![](/source/docs/assets/images/desk_images/247523.png)
-## Wait While It's Configured
-
-You will then have a short wait while Pantheon creates and allocates the resources for your site's environments. This takes only a few minutes under normal circumstances.
-![](/source/docs/assets/images/desk_images/247524.png)
-## Choose a Start State
+You can simply upload the archive file produced and Pantheon will do the rest.
+<div class="alert alert-danger" role="alert">
+<strong>Warning</strong>:  Archives cannot contain multiple <code>.sql</code> files, otherwise the import will fail.</div>
+
+
+## Manually Create Separate Site Archives
+
+Your codebase is required to import your site into Pantheon, as it will be used to create the initial code repository. This archive should include your entire WordPress codebase, including plugins, themes, configuration files etc.
+
+The code archive must include the following files and directories:
+```nohighlight
+├── index.php
+├── wp-activate.php
+├── wp-config.php
+├── wp-comments-post.php
+├── wp-blog-header.php
+├── wp-admin
+├── wp-cron.php
+├── wp-load.php
+├── wp-links-opml.php
+├── wp-includes
+├── xmlrpc.php
+├── wp-trackback.php
+├── wp-signup.php
+├── wp-settings.php
+├── wp-mail.php
+├── wp-login.php
+├── wp-content
+    ├── index.php
+    ├── mu-plugins
+    ├── themes
+    ├── plugins
+```
+### Create Code Archive
+The code archive should not include the `wp-content/uploads` or any other static assets that should not be tracked in version control. If your codebase contains static files they should be moved to the `wp-content/uploads` directory before export.
+
+```
+# Specify the destination folder.
+TARGET=~/Desktop
+# Specify the source folder.
+SOURCE=~/Projects/mysite
+# Change directory to the source folder.
+cd $SOURCE
+# Create an archive that excludes `wp-content/uploads`.
+tar -czf $TARGET/wordpress.tar.gz --exclude=wp-content/uploads* .
+```
+### Create WordPress Database Archive
+
+This is optional, but recommended. The easiest method is to use the [mysqldump](http://dev.mysql.com/doc/refman/5.5/en/mysqldump.html) utility to export your archive, then compress the result with gzip.
+```php
+# Specify the destination folder.
+TARGET=~/Desktop
+# Create the database backup.
+mysqldump -uUSERNAME -pPASSWORD DATABASENAME > $TARGET/db.sql
+# Compress the backup.
+gzip $TARGET/db.sql
+```
+### Export WordPress Files
+Export a tar.gz or .zip file of your files directory, which was intentionally omitted from the codebase import. These files are not tracked in Git; instead, they will be stored in Valhalla, our network file system.
+```php
+TARGET=~/Desktop
+SOURCE=~/Projects/mysite
+cd $SOURCE/wp-content/uploads
+tar -czf $TARGET/files.tar.gz .
+```
+
+
+
+## Import Via Terminus
+Using the following [Terminus](https://github.com/pantheon-systems/cli) command, you can import a single file archive in URL format that includes your codebase, database, and files:

```

Comment:
@bmackinney what kind of example archive do you want to see - blank WP site created on an alternate host? How should we link to that example other than providing a code block of the dir structure?